manim/utils/bezier.py

```diff
@@ -369,3 +372,114 @@ def get_quadratic_approximation_of_cubic(a0, h0, h1, a1):
 
 def is_closed(points: typing.Tuple[np.ndarray, np.ndarray]) -> bool:
     return np.allclose(points[0], points[-1])
+
+
+def bez_params_from_point(
+    point: typing.Iterable[typing.Union[float, int]],
+    control_points: typing.Iterable[typing.Iterable[typing.Union[float, int]]],
+) -> list[float]:
+    """Obtains the parameter corresponding to a given point
```

Comment:
I'm unsure of what "parameter" refers to here. Do you mean the t-value from 0 to 1 plugged into the bezier curve? because if so I'd think "the proportion along the bezier curve" would be better than "the parameter" (and then the function name should also be updated accordingly)

Reply:
I've renamed it to `proportion_along_bezier_curve_for_point`. Pretty long for my taste, but at least it's clearer.